card/mod/04_settings/set/self/recaptcha_private_key.rb

```diff
@@ -0,0 +1,10 @@
+event :update_recaptcha_private_key, :after=>:store do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.